ch_tools/monrun_checks_keeper/keeper_commands.py

```diff
@@ -26,15 +28,30 @@
 def alive_command(ctx):
     """Check (Zoo)Keeper service is alive"""
     try:
+        config = load_config()
```

Comment:
At this point the config should be already loaded and available in the context (`ctx.obj["config"]`).

Example of accessing config in another command:
https://github.com/yandex/ch-tools/blob/63be161b4a3fb3e45ef361100f8d25b0af92b5e2/ch_tools/chadmin/cli/object_storage_group.py#L45